cves/kernel/CVE-2016-6130.yml

```diff
@@ -55,15 +55,15 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: CVE-2016-6130 is a time of check time of use vulnerability. It occurred in a driver that interacts with the Linux kernel when a function requests the length of a response (time of check) and then receives that request (time of use). A malicious actor can use this time to change the length and cause information disclosure. This type of vulnerability is also commonly referred to as double-fetch.
```

Comment:
Nice description, I enjoyed that you described both parts of the vulnerability and clearly labeled what part you are describing.

Reply:
Thanks!